src/cascadia/WinRTUtils/inc/WtExeUtils.h

```diff
@@ -25,6 +25,19 @@ _TIL_INLINEPREFIX bool IsPackaged()
     return isPackaged;
 }
 
+_TIL_INLINEPREFIX uint8_t GetBranding()

```

Comment:
FWIW, I would much rather we not expose this broadly across the project -- especially with a generic name like this. **Nobody** should be calling this at runtime to make decisions based on branding, because that violates the rule that all branding-specific code should be compiled out when that branding isn't set.